src/movement.cpp

```diff
@@ -467,12 +469,12 @@ uint32_t MoveEvents::onItemMove(Item* item, Tile* tile, bool isAdd)
 	}
 
 	for (size_t i = tile->getFirstIndex(), j = tile->getLastIndex(); i < j; ++i) {
-		Thing* thing = tile->getThing(i);
+		auto thing = tile->getThing(i);
 		if (!thing) {
 			continue;
 		}
 
-		Item* tileItem = thing->getItem();
+		auto tileItem = thing->getItem();
```

Comment:
Could this be `const auto`
```suggestion
		const auto tileItem = thing->getItem();
```